ffmpeg/nodes.py

```diff
@@ -51,8 +52,21 @@ class FilterNode(Node):
     """FilterNode"""
     def _get_filter(self):
         params_text = self._name
-        arg_params = ['{}'.format(arg) for arg in self._args]
-        kwarg_params = ['{}={}'.format(k, self._kwargs[k]) for k in sorted(self._kwargs)]
+
+        # Helper function to escape uncomfortable characters
+        def escape_chars(seq, keys):
```

Comment:
If we pull this out to a global function we can put a test around it:
```
def _escape_chars(text):
    """Helper function to escape uncomfortable characters."""
    for ch in '\\[]=;:,':
        text = text.replace(ch, '\\' + ch)
    return text

# in test_ffmpeg.py:
def test_escape_chars():
    assert ffmpeg.nodes._escape_chars('a[b]c=d;e,\\f') == 'a\\[b\\]c\\=d\\;e\\,\\\\f'
```
Then `FilterNode._get_filter` becomes..
```
def _get_filter(self):
    params_text = self._name
    _args = [_escape_chars(str(x)) for x in self._args]
    _kwargs = {k: _escape_chars(str(v)) for k, v in self._kwargs.items()}
    ...
```